idmapping-common/src/main/java/org/uniprot/api/idmapping/common/request/uniprotkb/UniProtKBIdMappingGroupByRequest.java

```diff
@@ -0,0 +1,13 @@
+package org.uniprot.api.idmapping.common.request.uniprotkb;
+
+import org.springdoc.api.annotations.ParameterObject;
+
+import lombok.Data;
+
+@Data
+@ParameterObject
+public class UniProtKBIdMappingGroupByRequest {
+    private final String query;
+
+    private final String parentId;
+}
```

Comment:
We call parentId as "parent" in group by API. It is minor comment to keep the name consistent